it-and-security/lib/macos-check-if-apple-silicon.queries.yml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: query
 spec:
-  name: query_apple_silicon
+  name: Check if Mac has Apple silicon
   query: SELECT 1 FROM system_info WHERE cpu_type = "arm64e";
```

```suggestion
- name: Check if Mac has Apple silicon
   query: SELECT 1 FROM system_info WHERE cpu_type = "arm64e";
```

I think we need to match the GitOps format for queries. See this query as an example: https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/collect-failed-login-attempts.queries.yml

---

This does not work. See https://github.com/fleetdm/fleet/issues/20747

---

I was wrong. There's a bug: https://github.com/fleetdm/fleet/issues/20747